services/121-service/test/performance/helpers/performance.helper.ts

```diff
@@ -0,0 +1,420 @@
+import * as request from 'supertest';
+
+import { getPaymentSummary } from '@121-service/test/helpers/program.helper';
+import { getServer } from '@121-service/test/helpers/utility.helper';
+
+export type JsonRecord = Readonly<Record<string, unknown>>;
+
+export function jsonToCsv(data: JsonRecord[]): string {
+  if (!data || data.length === 0) return '';
+  const headers = Object.keys(data[0]);
+  const csvRows = [headers.join(',')];
+
+  for (const item of data) {
+    const row = headers.map((header) => {
+      const value = item[header];
+      return `"${String(value).replace(/"/g, '""')}"`;
+    });
+    csvRows.push(row.join(','));
+  }
+  return csvRows.join('\n');
+}
+
+// Helper function that waits and refreshes till the payment is processed or timeout occurs
+interface PaymentResultsParams {
```

Comment:
I think all these interfaces could be in separate files, or at least in one separate file?